sdk/core/azure-core/src/cryptography/md5.cpp

```diff
@@ -194,7 +194,6 @@ class Md5OpenSSL final : public Azure::Core::Cryptography::Hash {
     }
     if (1 != EVP_DigestInit_ex(m_context, EVP_md5(), NULL))
     {
-      EVP_MD_CTX_free(m_context);
```

Comment:
Context for why this is OK to remove:
https://github.com/Azure/azure-sdk-for-cpp/pull/3823/files#r926330079

> m_context is freed in the destructor for `Md5Hash`. This call does nothing.

Reply:
Btw, this is why using RAII types throughout the codebase is a good thing. Any time you see a `free` function outside a destructor should be considered a code smell.